framework/db/Command.php

```diff
@@ -780,6 +780,16 @@ public function execute()
 
             $this->pdoStatement->execute();
             $n = $this->pdoStatement->rowCount();
+            
+            // https://bugs.php.net/bug.php?id=61613
+            // Make sure that supported drivers throw an exception in case a statement of a multi-query failed
+            try {
+                while ($this->pdoStatement->nextRowSet()) {}
+            } catch (\Exception $e) {
+                if (strpos($e->getMessage(), 'SQLSTATE[IM001]') === false) {
```

Comment:
Maybe should use `getCode() === 'IM001'` instead though.
https://github.com/php/php-src/blob/0437aa2abf1eb494e4555d7d4dfbdbf566a59bdc/ext/pdo_mysql/tests/bug68371.phpt#L58